backend/modules/Pantheon/Providers/PantheonServiceProvider.php

```
@@ -0,0 +1,77 @@
+ * Wires the Pantheon connector into OpenMES.
+ *
+ * Registered only while the module is ENABLED (ModuleManager::loadEnabled), so an
+ * install without Pantheon carries no listener, no schedule entry and no menu
+ * item — and core keeps working exactly as before.
+ *
+ * The connector deliberately touches no core file: all Pantheon knowledge (view
+ * names, classification codes, document types, column mapping) lives in this
+ * module, and it feeds OpenMES through the same import services the ERP REST
+ * endpoints use.
+ */
+class PantheonServiceProvider extends ServiceProvider
```

PHPStan (2.2.7)

[error] Class Modules\Pantheon\Providers\PantheonServiceProvider extends unknown class Illuminate\Support\ServiceProvider.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(class.notFound)